.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.1.4
+    working_directory: ~/haul
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          - v1-dependencies
+      - run:
+          name: update-yarn
+          command: sudo npm i -g yarn
```

Comment:
An interesting idea would be to create a custom Docker image. The one I referenced is cool, but is nowhere documented (no repo that I could find). Might be nice to have it for future projects as well (will simplify infrastructure a lot)